src/main/java/AutomationPractice/ui/entities/Address.java

```diff
@@ -0,0 +1,330 @@
+/*
+ * @(#) FormPageFour.java Copyright (c) 2019 Jala Foundation.
+ * 2643 Av. Melchor Perez de Olguin, Colquiri Sud, Cochabamba, Bolivia.
+ * All rights reserved.
+ *
+ * This software is the confidential and proprietary information of
+ * Jala Foundation, ("Confidential Information").  You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jala Foundation.
+ */
+
+package AutomationPractice.ui.entities;
+
+import AutomationPractice.ui.StrategySetter;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Address class.
+ *
+ * @author Maday Alcala
+ * @version 0.0.1
+ */
+public class Address {
+    private String firstname;
+    private String lastname;
+    private String company;
+    private String address1;
```

Comment:
you can put a better name like mainAddress and secondAddress for address2

Reply:
Solved.